src/HealthChecks.UI/Extensions/EndpointRouteBuilderExtensions.cs

```diff
@@ -6,7 +6,8 @@
 
 namespace Microsoft.AspNetCore.Builder;
 
-public static class EndpointRouteBuilderExtensions
+// this type was renamed to avoid a name conflict with ASP.NET: https://github.com/dotnet/aspnetcore/issues/50836#issuecomment-1736455651
+public static class IEndpointRouteBuilderExtensions
```

Comment:
this is a workaround to avoid conflict with ASP.NET

when people use extension methods they typically don't specify the type name (the compiler does it for them) so it should not break most of the users